pages/content/pixi/index.md

```diff
@@ -0,0 +1,15 @@
+---
+$sitemap:
+  enabled: false
+globalText:
+  analyzeHeadline: Analyze your AMP page
```

Comment:
Should we nest instead of using a prefix?

```
text: 
  analyze:
    headline: Analyze...
    fieldPlaceholder: ...
```

Reply:
Done